packages/patterns/src/components/pattern-convert-button.js

```diff
@@ -6,6 +6,7 @@ import {
 	isReusableBlock,
 	createBlock,
 	serialize,
+	getBlockType,
```

Comment:
Should we use the `getBlockType` selector instead @gziolo ?

Reply:
Yes, that might be more appropriate. Although in practical terms the reactivity in the block more menu isn’t top priority as it’s a extremely rare case when the block type might change while selecting an action 😀